scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -284,7 +284,25 @@ def bicgstab(A, b, x0=None, tol=1e-5, maxiter=None, M=None, callback=None, atol=
                '``A`` must represent a hermitian, positive definite matrix.\n'
                'Alternatively, ``A`` can be a linear operator which can\n'
                'produce ``Ax`` using, e.g.,\n'
-               '``scipy.sparse.linalg.LinearOperator``.')
+               '``scipy.sparse.linalg.LinearOperator``.',
+               footer="""
+
+               Examples
+               --------
+               >>> from scipy.sparse import csc_matrix
+               >>> from scipy.sparse.linalg import cg
+               >>> import numpy as np
+               >>> R = np.random.randn(5,5)
+               >>> positive_definite = R @ R.T
+               >>> A = csc_matrix(positive_definite)
+               >>> b = np.random.randn(5)
+               >>> x, exitCode = cg(A,b)
```

Comment:
I know `exitCode` is already present in other examples, but it's pretty ugly. How about we clean that up while we're at it? Can you rewrite this line like so?:
```
>>> x, exit_code = cg(A, b)
```